pipeline_tools/pipelines/cellranger/cellranger.py

```diff
@@ -37,7 +37,9 @@ def create_cellranger_input_tsv(uuid, version, dss_url):
     # Parse inputs from metadata
     print('Gathering fastq inputs')
     fastq_files = [
-        f for f in primary_bundle.files.values() if f.file_format == 'fastq.gz'
+        f
+        for f in primary_bundle.files.values()
+        if f.file_format in ('fastq.gz', 'fastq')
```

Comment:
I don't know if this is necessary, but would doing something like 
```
if f.file_format.lower() in ('fastq.gz', 'fastq')
```
be useful to account for if a contributor puts "FASTQ" or "Fastq" for the format? I haven't specifically seen this before, but I'm trying to think ahead of the contributors :)

Reply:
Sure, `.lower()` is defo safer. Will do!